### DISCUSSION TITLE
Spring batch version 5.2

### DISCUSSION THREAD

**Discussion by Dimitrijed93** [spring-projects/spring-batch#4712] [Q&A] [answered]
Based on this page https://spring.io/projects/spring-batch#support

Version 5.2 should have been released yesterday. However that does not seem to be the case. Are there any information when we can expect the release?

**Comment by fmbenhassine** [comment, marked as answer]
We had an issue with Maven Central publishing last night and the 5.2 release did not complete as expected. We are working on it and we will send an update as soon as it is out. Thank you.

**Reply by Dimitrijed93** [reply]
Thanks!

**Reply by fmbenhassine** [reply]
FYI, the release is out now: https://github.com/spring-projects/spring-batch/releases/tag/v5.2.0 💪 🎆

**Reply by Dimitrijed93** [reply]
Great news! Many thanks!
